### DISCUSSION TITLE
Microsoft SSO 403 Forbidden

### DISCUSSION THREAD

**Discussion by ReinderVosDeWael** [open-webui/open-webui#6591] [Q&A] [answered]
I'm trying to run OpenWeb UI (hosted on Azure). I've passed the `MICROSOFT_CLIENT_ID`, `MICROSOFT_CLIENT_SECRET`, `MICROSOFT_CLIENT_TENANT_ID` as environment variables to the container, and doubly verified that these were copied correctly. I've also ensured that the redirect URI is set correctly.

When attempting to log in via Microsoft SSO, I get the following response:

```
{
    "detail": "You do not have permission to access this resource. Please contact your administrator for assistance."
}
```

Another resource, under the same app registration, works just fine. 

Any idea where to look for further debugging?

**Comment by ReinderVosDeWael** [comment, marked as answer]
Issue was related to registering via SSO (`ENABLE_OAUTH_SIGNUP`) not being set up via the environment variables.

**Reply by arunbugkiller** [reply]
HI @ReinderVosDeWael I have enabled "ENABLE_OAUTH_SIGNUP=true", and have also configured rest of the parameters right. I am still getting the same error. Any idea on how this can be resolved
